microprofile/tests/tck/tck-restfull/artifact-install.sh

```diff
@@ -0,0 +1,25 @@
+#!/usr/bin/env bash
+#
+# Copyright (c) 2023 Oracle and/or its affiliates. All rights reserved.
+#
+# This program and the accompanying materials are made available under the
```

Comment:
This is not a correct copyright for Helidon sources

Reply:
I guess I took it from somewhere in Helidon. I have taken other sh file as an example and updated the copyrights.